flake.nix

```diff
@@ -17,9 +17,10 @@
   }:
     flake-utils.lib.eachDefaultSystem (system: let
       pkgs = nixpkgs.legacyPackages.${system};
+      node = pkgs.nodejs_20;
       corepackEnable = pkgs.runCommand "corepack-enable" {} ''
```

Comment:
As it turns out the `corepack` package in `nixpkgs` already enables itself if included in the derivation, so if you're up for it @sectore you can just copy what we did in the `flake.nix` in the [`effect` repository](https://github.com/Effect-TS/effect/blob/main/flake.nix).

Reply:
Done 382024b . 

Not sure if `bun` is still needed here https://github.com/Effect-TS/website/blob/382024b31e0b1adea198fd8e1833b43c65d2bc7c/flake.nix#L18 There is a [pnpm-lock.yaml](https://github.com/Effect-TS/website/blob/main/pnpm-lock.yaml) in the repo only.  (just let me know)

Reply:
I believe both bun and nodePackages.json can be removed.

Reply:
Done 9e7068d